docs/getting-started/README.md

```diff
@@ -12,7 +12,7 @@ In order to install the `regen` binary, you'll need the following:
 - Make `>=4`
 - Go `>=1.15`
 
-For more information, see [Prerequisites](./prerequisites). 
+For more information, see [Prerequisites](prerequisites.md). 
```

Comment:
I think this works as intended actually, since the primary use of this document is not actually as a readme file, but as rendered by vuepress docs generator.


The link works as expected when viewed here: https://docs.regen.network/getting-started/